### ISSUE TITLE
Stuck at 20%[Bug]: 

### ISSUE THREAD

**Issue by Nyaruko166** [Griefed/ServerPackCreator#489]
### What happened?

It just stuck at 20% 
Both installer and portable have the same issue

### What did you expect to happen?

Let me in ;b

### Version

5.1.6

### Anything else you would like to add?

![image](https://github.com/Griefed/ServerPackCreator/assets/129191273/13360c67-33fc-4ac0-b17b-99a7fea0130a)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Griefed** [comment]
Heyo,

what OS are you running on?
Could you provide the logs from ~\ServerPackCreator\logs, if any, please?

**Comment by Nyaruko166** [comment]
![image](https://github.com/Griefed/ServerPackCreator/assets/129191273/2d865b56-1b63-4a48-8b19-e1519c8c28b6)

**Comment by Griefed** [comment]
> Could you provide the logs from ~\ServerPackCreator\logs, if any, please?

Sorry, `~\` indicates the user-home directory :D
For example: `C:\Users\<YourUserName>\ServerPackCreator\logs`

**Comment by Nyaruko166** [comment]
[serverpackcreator.log](https://github.com/Griefed/ServerPackCreator/files/13695516/serverpackcreator.log)

**Comment by Griefed** [comment]
Other people, myself included, are experiencing the same issue. This wasn't the case a couple of hours ago, so I guess something externally must've changed recently.

I'll get back to you when I know more.

**Comment by Griefed** [comment]
The NeoForge manifest is the culprit. A fix will be available in a few.

**Comment by Griefed** [comment]
```xml
<?xml version="1.0" encoding="UTF-8"?>
<metadata>
  <groupId>net.neoforged</groupId>
  <artifactId>forge</artifactId>
  <versioning>
    <latest>1.20.1-47.1.83</latest>
    <release>1.20.1-47.1.83</release>
    <versions>
      <version>1.20.1-47.1.7</version>
      <version>1.20.1-47.1.5</version>
      <version>1.20.1-47.1.8</version>
      <version>1.20.1-47.1.9</version>
      <version>1.20.1-47.1.11</version>
      <version>1.20.1-47.1.12</version>
      <version>1.20.1-47.1.23</version>
      <version>1.20.1-47.1.25</version>
      <version>1.20.1-47.1.26</version>
      <version>1.20.1-47.1.27</version>
      <version>1.20.1-47.1.28</version>
      <version>1.20.1-47.1.54</version>
      <version>1.20.1-47.1.55</version>
      <version>1.20.1-47.1.56</version>
      <version>1.20.1-47.1.57</version>
      <version>1.20.1-47.1.58</version>
      <version>1.20.1-47.1.59</version>
      <version>1.20.1-47.1.60</version>
      <version>1.20.1-47.1.61</version>
      <version>1.20.1-47.1.62</version>
      <version>1.20.1-47.1.63</version>
      <version>1.20.1-47.1.64</version>
      <version>1.20.1-47.1.65</version>
      <version>1.20.1-47.1.66</version>
      <version>1.20.1-47.1.67</version>
      <version>1.20.1-47.1.69</version>
      <version>1.20.1-47.1.70</version>
      <version>1.20.1-47.1.71</version>
      <version>1.20.1-47.1.72</version>
      <version>1.20.1-47.1.73</version>
      <version>1.20.1-47.1.74</version>
      <version>1.20.1-47.1.75</version>
      <version>1.20.1-47.1.76</version>
      <version>1.20.1-47.1.77</version>
      <version>1.20.1-47.1.78</version>
      <version>1.20.1-47.1.79</version>
      <version>1.20.1-47.1.80</version>
      <version>1.20.1-47.1.81</version>
      <version>47.1.82</version>
      <version>1.20.1-47.1.83</version>
    </versions>
    <lastUpdated>20231216215720</lastUpdated>
  </versioning>
</metadata>
```

One of these is not like the other.

**Comment by Griefed** [comment]
A workaround has been implemented in [5.1.7](https://github.com/Griefed/ServerPackCreator/releases/tag/5.1.7)
